.github/workflows/avm.platform.toggle-avm-workflows.yml

```
@@ -0,0 +1,48 @@
+name: 'avm.platform.toggle-avm-workflows'
+
+on:
+  workflow_dispatch:
+    inputs:
+      workMode:
+        type: choice
+        description: "Enable or disable workflows"
+        required: true
+        options:
+          - "enable"
+          - "disable"
+        default: "disable"
+      includePattern:
+        type: string
+        description: "RegEx which workflows are included"
+        required: false
+        default: "avm.res"
```

```suggestion
        default: "avm\.(?:res|ptn)"
```
Also note that `.` is a special char in regex ;)